jobs/build/appregistry/Jenkinsfile

```diff
@@ -71,6 +72,18 @@ node {
                         description: '(Optional) List of images to limit selection (default all)',
                         defaultValue: ""
                     ),
+                    [
+                        name: 'STREAM',
+                        description: 'OMPS appregistry',
+                        $class: 'hudson.model.ChoiceParameterDefinition',
+                        choices: ['dev', 'stage', 'prod'].join('\n'),
```

Comment:
I know this is just `WIP` now but as of `2.121` we can stop doing the `list` -> `join` process: https://jenkins.io/doc/upgrade-guide/2.121/ (on the bottom for 2.121.1)

> Choice parameters now allow the use of a list of options instead of a string of options separated by line breaks.

Reply:
that's great! looked like a workaround, going to fix that now ;D